### PR TITLE
Update and improve issue template

### PR DESCRIPTION
**Pull request by bjoernricks**
Always link to community platform. Be more stricter about not using the
issue tracker as support forum.

Add some notes for adding content to the sections.

### CODE REVIEW

**Comment by bjoernricks** [comment]
@janowagner  @cfi-gb @swaterkamp  Can I get some feedback for this changes? If not I am going to merge this at tomorrow as it is.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -3,23 +3,49 @@ If reporting an issue please try to provide the information asked below.
 
 Before reporting an issue please:
 
-1. make sure that you're using the latest published GVM components for the
-   release you are using: http://openvas.org/install-source.html.
-2. check the list of issues whether it isn't already reported.
-3. be aware that this is not a support forum. If your issue is more a question
-   than a bug report, please use https://community.greenbone.net/c/gse instead.
+1. be aware that this is not a support forum. If your issue is more a question
```
Comment (swaterkamp) [inline .github/ISSUE_TEMPLATE.md]: I'd use capital letters for the first word at all bullet points.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -3,23 +3,49 @@ If reporting an issue please try to provide the information asked below.
...
+  Is this really an bug? If in doubt please use
+  https://community.greenbone.net/c/gse instead.
+  Please attach screenshots if possible! They are extremely helpful for
+  diagnosing issues.
+-->
 
 ### Steps to reproduce
+
+<!--
+  How would you describe your issue to someone who doesn’t know GSA.
```
Comment (swaterkamp) [inline .github/ISSUE_TEMPLATE.md]: Use a ? instead of . This is question ;)

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (swaterkamp) [reply .github/ISSUE_TEMPLATE.md]: Maybe rephrase to "If your issue is rather a question than a bug report [...]".

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -3,23 +3,49 @@ If reporting an issue please try to provide the information asked below.
...
-2. check the list of issues whether it isn't already reported.
-3. be aware that this is not a support forum. If your issue is more a question
-   than a bug report, please use https://community.greenbone.net/c/gse instead.
+1. be aware that this is not a support forum. If your issue is more a question
+   than a bug report, please use our community forum at
+   https://community.greenbone.net/c/gse instead.
+2. make sure that you're using the latest published GVM components for the
+   release you are using: https://community.greenbone.net/t/about-the-source-edition-gse-category/176
+3. check the list of issues whether it isn't already reported.
+4. read 1. again and if you still belive you found a software bug please keep
```
Comment (swaterkamp) [inline .github/ISSUE_TEMPLATE.md]: ```*``` believe

Comment (swaterkamp) [reply .github/ISSUE_TEMPLATE.md]: ```*``` please continue to file the this issue.
